src/Illuminate/Support/Collection.php

```diff
@@ -296,6 +296,36 @@ public function containsStrict($key, $value = null)
         return in_array($key, $this->items, true);
     }
 
+    /**
+     * Join all items from the collection using a string. The final items can use a separate glue string.
+     *
+     * @param  string  $glue
+     * @param  string  $finalGlue
+     * @return string
+     */
+    public function join($glue, $finalGlue = '')
+    {
+        if ($finalGlue === '') {
```

Comment:
Should we change the default value to `null` instead of `''`, as this removes the option for having an empty string as your `finalGlue`.